### ISSUE TITLE
Improvement for the doc Certificate Guide - Add RA and CA defintion 

### ISSUE THREAD

**Issue by josenogueiradalmeida** [OpenBanking-Brasil/specs-seguranca#23]
In the doc [Certificate Guide](https://github.com/OpenBanking-Brasil/specs-seguranca/blob/main/open-banking-brasil-ICP-certificate-guide.pdf), it would help to remember the very basic definition for RA and CA, as well the difference between them.

For instance: (from wikipedia)
The PKI role that may be delegated by a CA to assure valid and correct registration is called a registration authority (RA). 
Basically, an RA is responsible for accepting requests for digital certificates and authenticating the entity making the request.

**Comment by RalphBragg** [comment]
The PDF has been created by the ICP and Security Working Groups. Ideally a mark down version of the document would be created so that it can be worked on and improved. Marcos is not currently active on this git but it would be useful if he could be.

**Comment by DeiGratia33** [comment]
The certificate DOC uploaded to the github will not be used as guidance. Please refer to the padrao_certificados.md as future guidance for Certificates.

**Comment by RalphBragg** [comment]
Resolved with MD guidance being added.
